docs/content/en/schemes/local.md

```diff
@@ -60,6 +60,29 @@ export default {
 </script>
 ```
 
+## Backend
+
+You'll need a backend server that implement the basics of authentication. As
+this is very security-sensitive code, we strongly recommend that you use an
+existing authentication backend library.
+
+The backend should verify the login credentials, then return a JSON body with
+the token that the frontend can use to act as this user. The JSON body format
+is configured in the [token section of the local scheme configuration](#token).
+
+The entire backend response is passed through to the `loginWith` response,
+so you can pass through additional information about the user, e.g. for
+authorization (which is out of scope of @nuxtjs/auth).
+
+If [`user.autoFetch` is true (default)](#user), then a request to
+`endpoints.user` immediately after a successful login. That endpoint should
+respond with the JSON information for a specific user. That information is
+assigned directly to [the user property](../api/auth#user).
+
+If you'd prefer to return the user's information directly from the login
+session, configure `user.autoFetch` to true, fetch the user information from the
+`loginWith` response, and pass it in to
+[`setUser`](../api/auth#setuseruser).
```

Comment:
Maybe we can also add a note here saying that user endpoint still needed, since user data is not stored.

Reply:
Thanks, this is something I don't know about. When would that be used?

Reply:
The use case for `autoFetch` is exactly the one you described above. If the user data is already included in the login response, you can disable the user fetch that happens right after login. But you still need the user endpoint to fetch the user on reload pages.

So I think the note could be something like this:

> You still need to set the user endpoint in order to fetch the user when page is reloaded, since we do not store the user data.

Reply:
Is there any way for this module to be implemented without adding the user info endpoint? Is it a strict requirement? Why?

Reply:
> Is there any way for this module to be implemented without adding the user info endpoint?

Actually, yes, you need to set user endpoint to `false`. Then the user will be set to `{}`.

But what I wanted to say is that sometimes people misunderstand what `autoFetch` does. So a warn that user endpoint still needed in order to have user data.

Reply:
Pushed some clarifications. It's maybe a bit repetitive but hopefully addresses your point. What do you think?

Reply:
Awesome! Nice work! :smiley: